edb/server/protocol/auth_ext/ui.py

```diff
@@ -36,6 +36,35 @@
 ]
 
 
+def _render_oauth_buttons(
+    oauth_providers: list,
+    oauth_params: dict[str, str],
+    oauth_label: str,
+) -> str:
+    if not oauth_providers:
+        return ""
+
+    oauth_buttons = [
+        {
+            "authparams": urllib.parse.urlencode(
+                {'provider': p.name, **oauth_params}
+            ),
+            "image": (
+                f'<img src="_static/icon_{p.name[15:]}.svg"'
+                f'alt="{p.display_name} Icon" />'
```

Comment:
We need a space here, not sure which makes more sense:

```suggestion
                f'<img src="_static/icon_{p.name[15:]}.svg" '
                f'alt="{p.display_name} Icon" />'
```

or

```suggestion
                f'<img src="_static/icon_{p.name[15:]}.svg"'
                f' alt="{p.display_name} Icon" />'
```

Reply:
Fixed.